### ISSUE TITLE
Feature flag service build fails on Mac M1

### ISSUE THREAD

**Issue by mviitane** [open-telemetry/opentelemetry-demo#901]
# Bug Report

Which version of the demo you are using?
Using main/latest

## Symptom
Starting with a clean system
`$ docker system prune -a`

Build fails:
```
$ docker compose build featureflagservice
…
 => ERROR [builder 20/25] RUN ~/.mix/rebar3 grpc_regen                                                                                                                                                                                                      0.3s
------
 > [builder 20/25] RUN ~/.mix/rebar3 grpc_regen:
#0 0.236 escript: exception error: undefined function rebar3:main/1
#0 0.236   in function  escript:run/2 (escript.erl, line 758)
#0 0.236   in call from escript:start/1 (escript.erl, line 277)
#0 0.236   in call from init:start_em/1 
#0 0.236   in call from init:do_boot/3 
------
failed to solve: executor failed running [/bin/sh -c ~/.mix/rebar3 grpc_regen]: exit code: 127
```

**Comment by tsloughter** [comment]
Is there anything cached? That error usually means the `rebar3` is built for an incompatible version of Erlang.

**Comment by tsloughter** [comment]
I can confirm it fails for me too the same way and I'm not on a Mac. So this is unrelated to the arch.

**Comment by tsloughter** [comment]
Oooooh, I know what it is.

We released rebar3 3.21.0 last week and it is built for OTP-26 (we only support the last 3 versions of OTP, so 23 which this Dockerfile uses is no longer supported -- well it technically works, but not with the prebuilt binary that the Dockerfile downloads).

**Comment by mviitane** [comment]
This got fixed with #903
